README.md

```diff
@@ -92,6 +95,18 @@ mixpanel.token = myapp.mixpanel.dinamic_token
 For view code dealing with requests, a pre-configured `request.mixpanel`
 is available.
 
+## User configuration
+
+A `user` object needs to have following attributes:
+  * distinct_id: str
+  * email: str
+  * name: str
+  * created: datetime.datetime
+  * state: str
+
+If `mixpanel.no_initial_user` is true then `request.mixpanel.user` needs to be initialized with User object along with at least `distinct_id` attribute.
+
+`request.mixpanel.profile_sync()` can be used to create/update profile.
```

Comment:
I'm thinking I'd rather throw out `profile_sync`. Then a bunch of assumptions go away, right?

Reply:
Yes, that can be done and we can use `profile_set` instead of profile_sync to update the user properties

This also removes the dependency on user email.

Reply:
> we will lose backward compatibility

Only Kai currently using this lib. And we don't have paid users on Kai yet. So meh, break it.

Reply:
ok. I will also try to include people_append method in it.